bluez-async/src/macaddress.rs

```diff
@@ -0,0 +1,123 @@
+use std::convert::TryInto;
+use std::fmt::{self, Debug, Display, Formatter, LowerHex, UpperHex};
+use std::str::FromStr;
+use thiserror::Error;
+
+/// An error parsing a MAC address from a string.
+#[derive(Clone, Debug, Error, Eq, PartialEq)]
+#[error("Invalid MAC address '{0}'")]
+pub struct ParseMacAddressError(String);
+
+/// MAC address of a Bluetooth device.
+#[derive(Clone, Copy, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub struct MacAddress([u8; 6]);
```

Comment:
need at least an accessor for this if you want to impl from MacAddress anywhere efficiently

Reply:
Done.